.idea/prettier.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was this intended?

Reply:
I've seen that `.idea` is not included in `.gitignore` so i kept it, but I can remove if may cause errors 👀

Reply:
Not sure, but it becomes a personal overwrite everytime then.
Maybe better we add it to gitignore :)